src/java/org/apache/cassandra/schema/MigrationCoordinator.java

```diff
@@ -355,11 +356,11 @@ private synchronized void removeEndpointFromVersion(InetAddressAndPort endpoint,
         FutureTask<Void> task = new FutureTask<>(() -> pullSchema(new Callback(endpoint, info)), null);
         if (shouldPullImmediately(endpoint, info.version))
         {
-            Stage.MIGRATION.submit(task);
+            MIGRATION.submit(task);
```

Comment:
nit: tbh, I kind of like the readability properties of the `Stage.X` here...also cuts down on the diff...

Reply:
We do not have a code style guide to address the static import, and admittedly there are multiple styles of such case in the code base. (might worthy to have a broad discussion in dev thread)

I'd prefer to just follow the existing style of the file... also agree with Caleb, `Stage.X` provides better readability.

Reply:
Yeah, I saw the same, sounds like a good idea to address it, I can add the topic to my list of things I want to address after we close the current bigger topics on the mailing list